pobieracz_danych_gugik.py

```diff
@@ -1006,7 +1002,7 @@ def bdot_selected_powiat_btn_clicked(self):
         if not powiatName:
             self.no_area_specified_warning()
             return
-        teryt = self.dockwidget.regionFetch.getTerytByPowiatName(powiatName)
+        teryt = self.dockwidget.powiat_cmbbx.currentData()
```

Comment:
Prawdopodobnie po tych zmianach dzieje się to przy pobieraniu BDOT 10 K dla powiatu:
![image](https://github.com/envirosolutionspl/pobieracz_danych_gugik/assets/113895660/d6a226b1-2eb1-457c-bef6-88082b98072d)